examples/nextjs-with-supabase-auth/pages/profile.js

```diff
@@ -1,13 +1,8 @@
 // pages/profile.js
-import { withAuthRequired } from '@supabase/supabase-auth-helpers/nextjs'
+import { withPageAuth } from '@supabase/auth-helpers-nextjs';
 
 export default function Profile({ user }) {
-  return (
-    <>
-      <div>Hello {user.email}</div>
-      <pre>{JSON.stringify(user, null, 2)}</pre>
```

Comment:
Any particular reason why the pre tag and its content was removed?

Reply:
It was done out of testing, now back in the code.